astropy/io/fits/convenience.py

```diff
@@ -709,6 +712,110 @@ def info(filename, output=None, **kwargs):
     return ret
 
 
+def quickdiff(inputa, inputb, *args, **kwargs):
+    """
+    Compare two parts of a fits file, including entire FITS files, and
+    FITS HDU objects.
+
+    Parameters
+    ----------
+    inputa : file path, file object, or file like object
+        The filename of a FITS file on disk, or fits file object
+        to compare to inputb.
+
+    inputb : file path, file object, or file like object
+        The filename of a FITS file on disk, or a fits file object
+        to compare to inputa.
+
+    ext
```

Comment:
There is no actual `ext`-parameter in the function signature. Maybe better to name this `ext, extname, extver` because these are the parameters that are looked for in the `kwargs`.